api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -127,6 +126,22 @@ public Workspace get(String ns, String firecloudName) {
                 fcWorkspace -> fcWorkspace));
   }
 
+  @Override
+  public Map<User, WorkspaceAccessEntry> getFirecloudWorkspaceAcls(Workspace workspace) {
```

Comment:
Lets simplify this method to take just the namespace and firecloud name rather than the full workspace.